modules/table/src/main/java/org/apache/ignite/internal/table/distributed/TableManager.java

```diff
@@ -1459,12 +1467,6 @@ private CompletableFuture<Void> destroyTableLocally(int tableId) {
         InternalTable internalTable = table.internalTable();
         int partitions = internalTable.partitions();
 
-        // TODO https://issues.apache.org/jira/browse/IGNITE-18991 Move assigment manipulations to Distribution zones.
-        Set<ByteArray> assignmentKeys = IntStream.range(0, partitions)
-                .mapToObj(p -> stablePartAssignmentsKey(new TablePartitionId(tableId, p)))
-                .collect(toSet());
-        metaStorageMgr.removeAll(assignmentKeys);
```

Comment:
Why we don't remove assignment keys now?

Reply:
It has to be done by @alievmirza.
Can you explain where we are removing stable assignment. I am sure it should be done when a zone is removed.

Reply:
I created another ticket.
https://issues.apache.org/jira/browse/IGNITE-22043